example-playbooks/restore_scylla_manager_backup/restore.yaml

```diff
@@ -50,7 +45,12 @@
         line: "initial_token: {{ _initial_token.stdout }}"
         create: yes
 
-    - name: Make each node a seed
+    - name: Get seeds list
+      shell: |
+        grep '\- seeds:' /etc/scylla/scylla.yaml | awk '{ print $NF }' | tr -d '"' | tr -d "'"
```

Comment:
Where seeders config got into  `/etc/scylla/scylla.yaml` from?

Do you assume that Scylla "new" cluster has been previously properly provisioned?

If yes, then, please, add this to this example description.
It seems that the description is completely missing - please, add something:
 * Describe the prerequisites.
 * Describe the algorithm.

Reply:
Yes I'm taking a new cluster from Scylla Cloud.

Reply:
But all data dirs are wiped out so it's like a fresh cluster.